sw/device/lib/base/memory_perftest.c

```diff
@@ -13,7 +13,10 @@
 #include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 
-#define PERFTEST_BUF_LEN 1000
+enum {
+  kBufLen = 1000,
```

Comment:
> Oh neat, I guess defining this in an enum enables us to use the value statically, like C++'s constexpr. I didn't understand that trick until now!

Yeah, this is our preferred style. Please also note that we don't name the `enum` if we don't need to refer to its type and use it just to declare some constants as in this case.

> Could this trick possibly introduce an implicit narrowing conversion for values outside the range of the enum's underlying type? (I'm not worried about the usage in this PR, I'm certain that 1000 will fit in `int`.)

No because the underlying type of the `enum`, whatever it is, must be capable of representing all enumerator values of that enumeration.